frontend2/src/utils/types/.openapi-generator/FILES

```diff
@@ -1,65 +1,64 @@
-api/ApiApi.ts
-api/api.ts
-configuration.ts
-git_push.sh
+.openapi-generator-ignore
+apis/ApiApi.ts
+apis/index.ts
 index.ts
-model/Autoscrim.ts
-model/ClassRequirement.ts
-model/CountryEnum.ts
-model/EligibilityCriterion.ts
-model/Email.ts
-model/Episode.ts
-model/GenderEnum.ts
-model/HistoricalRating.ts
-model/LanguageEnum.ts
-model/Match.ts
-model/MatchParticipant.ts
-model/MatchReport.ts
-model/ModelMap.ts
-model/PaginatedClassRequirementList.ts
-model/PaginatedEpisodeList.ts
-model/PaginatedMatchList.ts
-model/PaginatedScrimmageRequestList.ts
-model/PaginatedSubmissionList.ts
-model/PaginatedTeamPublicList.ts
-model/PaginatedTournamentList.ts
-model/PaginatedTournamentRoundList.ts
-model/PasswordToken.ts
-model/PatchedTeamPrivate.ts
-model/PatchedUserPrivate.ts
-model/PlayerOrderEnum.ts
-model/ReleaseStatusEnum.ts
-model/ResetToken.ts
-model/SaturnInvocation.ts
-model/ScrimmageRequest.ts
-model/ScrimmageRequestStatusEnum.ts
-model/Status526Enum.ts
-model/StatusBccEnum.ts
-model/StyleEnum.ts
-model/Submission.ts
-model/SubmissionDownload.ts
-model/SubmissionReport.ts
-model/TeamAvatar.ts
-model/TeamCreate.ts
-model/TeamJoin.ts
-model/TeamPrivate.ts
-model/TeamProfilePrivate.ts
-model/TeamProfilePublic.ts
-model/TeamPublic.ts
-model/TeamReport.ts
-model/TokenObtainPair.ts
-model/TokenRefresh.ts
-model/TokenVerify.ts
-model/Tournament.ts
-model/TournamentRound.ts
-model/TournamentSubmission.ts
-model/UserAvatar.ts
-model/UserCreate.ts
-model/UserPassed.ts
-model/UserPrivate.ts
-model/UserProfilePrivate.ts
-model/UserProfilePublic.ts
-model/UserPublic.ts
-model/UserResume.ts
-model/models.ts
-variables.ts
+models/Autoscrim.ts
+models/ClassRequirement.ts
+models/CountryEnum.ts
```

Comment:
Just to confirm (in similar vein to other comment) that this folder structure change will persist forever after generate types is run so that no problems pop up

Reply:
yup, I made no changes to the /types folder after generating